data_collection/gazette/spiders/go/go_valparaiso_de_goias.py

```diff
@@ -0,0 +1,14 @@
+import datetime as dt
+
+from gazette.spiders.base.nucleogov import NucleoGovGazetteSpider
+
+
+class GoValparaisoDeGoiasSpider(NucleoGovGazetteSpider):
+    name = "go_valparaiso_de_goias"
+    TERRITORY_ID = "5221858"
+    allowed_domains = [
+        "diariooficial.valparaisodegoias.go.gov.br",
+    ]
+    url_base = "https://diariooficial.valparaisodegoias.go.gov.br/api/diarios?data={}&calendar=true&situacao=2"
+
```

Comment:
Reflete aqui a mudança na classe base

```suggestion
    url_base = "https://diariooficial.valparaisodegoias.go.gov.br"
```